examples/opengl_overlay_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+# SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+cmake_minimum_required(VERSION 3.14)
+project(opengl_overlay_cpp LANGUAGES CXX)
```

Comment:
Can we remove the _cpp from the example name.  (Also in the directory name)

Reply:
The idea is that we could have both rust and cpp code for the same demo  (like we have for other demos)

Reply:
This is done now.